src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2866,16 +2866,18 @@ because xbuild doesn't support framework reference assemblies.
   </BuildApk>
 </Target>
 
-<PropertyGroup>
-	<_CopyPackageInputs>
-		$(MSBuildAllProjects)
-		;@(ReferencePath)
-		;@(ReferenceDependencyPaths)
-		;$(ApkFileIntermediate)
-		;$(_AndroidBuildPropertiesCache)
-		;@(ApkFiles)
-	</_CopyPackageInputs>
-</PropertyGroup>
+<Target Name="_ResolveCopyPackageInputs">
+	<PropertyGroup>
+		<_CopyPackageInputs>
+			$(MSBuildAllProjects)
```

Comment:
Can we remove the `$(MSBuildAllProjects)` here? If important project settings change then `$(_AndroidBuildPropertiesCache)` would change?

I’ve been wanting to go through and remove all these if you think it is a good idea. It would prevent MSBuild from reading the time stamps on several files in each instance. We have this in a lot of places, so it might be noticeable in a build with no changes.

Reply:
done